WordPress/Info.plist

```diff
@@ -8,96 +8,85 @@
 	<string>${PRODUCT_NAME}</string>
 	<key>CFBundleExecutable</key>
 	<string>${EXECUTABLE_NAME}</string>
-	<key>CFBundleIcons~ipad</key>
+	<key>CFBundleIcons</key>
```

Comment:
I've rolled back all changes to this Info.plist since none of these changes done manually.  I believe the one to trigger all the changes was the `NSUserActivityTypes` one.

I've re-tested my PR without that change and it seems to work just fine.  Can I ask you to double check this @frosty ?